lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -562,6 +575,7 @@ def __init__(self):
         self.replicas = -1
         self.service_id = False
         self.service_version = False
+        self.resolve_image = None
```

Comment:
I don't think it makes sense to have `resolve_image` here, as it is nothing passed to docker.

Reply:
You also assign to it in `from_ansible_params()`.

Reply:
Very true.